src/selfplay/loop.cc

```diff
@@ -24,6 +24,9 @@
   terms of the respective license agreement, the licensors of this
   Program grant you additional permission to convey the resulting work.
 */
+#include <sstream>
+#include <iostream>
+#include <iomanip>
```

Comment:
This is unnecessary, please remove this file from the diff.

Reply:
done